### DISCUSSION TITLE
Reduce the size of log.

### DISCUSSION THREAD

**Discussion by alvaroalea** [photoview/photoview#399] [General]
There is a easy way to reduce the log verbose?
Every time (every 5 minutes in my config) the scan process dump a lot of lines in the docker log for every photo, already in de db or not. so in few days the log file of the docker /var/lib/docker/containers/lotsofnumbers/lotsofnumbers.log grown until use the complete partition.
like:

"log":"2021/06/07 21:03:55 Process media transaction enter (/photo.......
{"log":"2021/06/07 21:03:55 Processing photo: /photos/c.......
{"log":"2021/06/07 21:03:55 Process media transaction exit (/photo......

or 

{"log":"2021/06/07 20:13:19 Process media transaction enter (/pho......
{"log":"2021/06/07 20:13:19 Processing photo: /phot.....
{"log":"2021/06/07 20:13:19 Media already scanned: /phot.....
{"log":"2021/06/07 20:13:19 Process media transaction exit (/pho....

**Comment by viktorstrate** [comment]
There are probably some cleanup that could be done, but I think the best solution would be to split logging into categories based on importance, like `DEBUG`, `INFO`, `NOTICE` `WARNING`, `ERROR`, then let the user specify the amount of logging.
I could imagine that a lot of great libraries exists to do this.
